ui/src/SampleUpload.js

```diff
@@ -47,6 +47,19 @@ class SampleUpload extends Component {
       return;
     }
 
+    // Comment here to explain why 1,000,000 and not 1024*1024.
+    // Only dividing by 1,000,000 gives the size in mb that agrees with value on mac
+    var file_size_in_mb = e.target.files[0].size / 1000000;
+
+    if (file_size_in_mb > 100) {
+      alert(
+        "Maimum file size is 100mb.  This file size is: " +
+          file_size_in_mb +
+          " mb"
```

Comment:
Is it possible to make this limit configurable from the environment?

Reply:
I'm not sure of the best way of doing this,  I'd want it to be shared correctly with the support tool java app - set once in K8s, docker-dev etc.
After an initial google I'm not seeing an initial nice way.
One possibility is to create a new endpoint within the Java endpoint, this would expose the max file size settings back to the React.js app.